Ui/Component/Listing/Column/NotificationColumn.php

```diff
@@ -0,0 +1,128 @@
+ *        ######  #####( ######  #####. ######  #####  ######  #####  ######
+ * ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+ * ###### ######  #####( ######  #####. ######  #####          #####  ######
+ * #############  #############  #############  #############  #####  ######
+ *  ############   ############  #############   ############  #####  ######
+ *                                      ######
+ *                               #############
+ *                               ############
+ *
+ * Adyen Payment module (https://www.adyen.com/)
+ *
+ * Copyright (c) 2019 Adyen BV (https://www.adyen.com/)
+ * See LICENSE.txt for license details.
+ *
+ * Author: Adyen <magento@adyen.com>
+ */
+
+namespace Adyen\Payment\Ui\Component\Listing\Column;
+
+class NotificationColumn extends \Magento\Ui\Component\Listing\Columns\Column
+{
+
+    /**
+     * @var \Magento\Sales\Api\Data\OrderInterface
+     */
+    protected $orderInterface;
+
+    /**
+     * @var \Magento\Backend\Helper\Data
+     */
+    protected $backendHelper;
+
+    /**
+     * @var \Adyen\Payment\Helper\Data
+     */
+    protected $adyenHelper;
+
+    public function __construct(
+        \Magento\Sales\Api\Data\OrderInterface $orderInterface,
+        \Magento\Backend\Helper\Data $backendHelper,
+        \Adyen\Payment\Helper\Data $adyenHelper,
+        \Magento\Framework\View\Element\UiComponent\ContextInterface $context,
+        \Magento\Framework\View\Element\UiComponentFactory $uiComponentFactory,
+        array $components = [],
+        array $data = []
+    ) {
+        $this->orderInterface = $orderInterface;
+        $this->backendHelper = $backendHelper;
+        $this->adyenHelper = $adyenHelper;
+        parent::__construct($context, $uiComponentFactory, $components, $data);
+    }
+
+
+    /**
+     * Style and format Adyen notification columns
+     *
+     * @param array $dataSource
+     * @return array
+     */
+    public function prepareDataSource(array $dataSource)
+    {
+
+        if (isset($dataSource['data']['items'])) {
+
+            foreach ($dataSource['data']['items'] as & $item) {
+
+                //Setting success column CSS class
+                switch ($item["success"]) {
+
+                    case "true";
+                        $class = "grid-severity-notice";
+                        break;
+                    case "false";
+                        $class = "grid-severity-critical";
+                        break;
+                    default:
+                        $class = "grid-severity-minor";
+                        break;
+
+                }
+                $item["success"] = sprintf('<span class="%s">%s</span>', $class, $item["success"]);
+
+                //Setting Status "fake" column value based on processing and done values
+                if ($item["processing"] == 0) {
+                    if ($item["done"] == 0) {
+                        $item["status"] = "Queued";
```

Comment:
Can you please use translations for these values?

Reply:
Phrase translations added